lighthouse-core/report/html/renderer/dom.js

```diff
@@ -126,7 +129,21 @@ class DOM {
         a.rel = 'noopener';
         a.target = '_blank';
         a.textContent = linkText;
-        a.href = (new URL(linkHref)).href;
+
+        const url = new URL(linkHref);
+
+        if (url.origin === DEVELOPERS_GOOGLE_ORIGIN) {
```

Comment:
I wonder if there's any reason not to include these UTM params on every outgoing link.

Reply:
I don't think there is a good reason to share that data with all websites (some of which are 3rd party), but also I'd like to avoid interfering with UTM params that we already send to e.g. axe docs. If there are other origins we should put on a safelist, besides developers.google.com, I'll be happy to do that.